test/org/opensolaris/opengrok/web/SearchHelperTest.java

```diff
@@ -18,18 +18,159 @@
  */
 
 /*
- * Copyright (c) 2012, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2016, Oracle and/or its affiliates. All rights reserved.
  */
 
 package org.opensolaris.opengrok.web;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeSet;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
+import org.opensolaris.opengrok.configuration.RuntimeEnvironment;
+import org.opensolaris.opengrok.index.Indexer;
+import org.opensolaris.opengrok.index.IndexerTest;
+import org.opensolaris.opengrok.search.QueryBuilder;
+import org.opensolaris.opengrok.util.TestRepository;
 
 /**
  * Unit tests for the {@code SearchHelper} class.
  */
 public class SearchHelperTest {
+    TestRepository repository;
+    private final String ctagsProperty = "org.opensolaris.opengrok.analysis.Ctags";
+    RuntimeEnvironment env;
     
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+    }
+
+    @Before
+    public void setUp() throws IOException {
+        repository = new TestRepository();
```

Comment:
we also duplicate this on quite a few places, dedup might be nice (but for this dup not in this changeset, just for future)